package.json

```diff
@@ -51,6 +51,7 @@
     "ts-node": "~8.3.0",
     "tslint": "~6.1.0",
     "typescript": "~4.0.2",
-    "@microsoft/applicationinsights-properties-js": "2.5.11"
+    "@microsoft/applicationinsights-properties-js": "^2.6.0",
+    "@microsoft/applicationinsights-analytics-js": "^2.6.0"
```

Comment:
This is for testing purpose only, because in the test it needs to manually new appInsights instance with ApplicationInsights type imported from analytics plugin.